test/remoting.integration.js

```diff
@@ -118,13 +118,14 @@ describe('remoting - integration', function() {
         'find(filter:object):store GET /stores',
         'findOne(filter:object):store GET /stores/findOne',
         'updateAll(where:object,data:object) POST /stores/update',
-        'deleteById(id:any) DELETE /stores/:id',
+        'deleteById(id:any):number DELETE /stores/:id',
```

Comment:
OK, I'm going to update everything according to your feedback and reassign to you for review after.